src/data/staking-products.json

```
@@ -250,6 +279,32 @@
         "eventAction": "Clicked",
         "eventName": "Clicked ethdo go to link"
       }
+    },
+    {
+      "name": "AVADO",
+      "svgPath": "avado-glyph.svg",
+      "hue": 239,
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
+      "isFoss": true,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "isSelfCustody": true,
+      "platforms": ["Linux", "macOS", "Windows", "Browser"],
```

```suggestion
      "platforms": ["Browser"],
```

Changing this for consistency as well